packages/blocks/src/note-block/note-block.ts

```diff
@@ -17,10 +17,7 @@ export class NoteBlockComponent extends BlockElement<NoteBlockModel> {
     }
   `;
 
-  override connectedCallback() {
-    super.connectedCallback();
-    bindHotKey(this);
-  }
+  keymapController = new KeymapController(this);
```

Comment:
Should we further promote this pattern of removing setups in `connectedCallback`?

Reply:
I think it's a good pattern for sharing logic in lit. I think it's better than maintain lifecycle in `xxxManager` because it can share the life cycle with lit element.